src/main/java/com/snack/news/controller/SlackBotController.java

```diff
@@ -0,0 +1,35 @@
+package com.snack.news.controller;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.snack.news.exception.SlackAuthorizationException;
+import com.snack.news.service.SlackBotService;
+import com.snack.news.util.SlackAuthHttpClient;
+import lombok.AllArgsConstructor;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid unused imports such as 'org.springframework.web.bind.annotation'](https://app.codacy.com/manual/snack-news/Snack-BE/pullRequest?prid=5471424)